File: BIPServer/src/com/ccf/bip/biz/system/monitor/mapper/ServerStatus.java
package com.ccf.bip.biz.system.monitor.mapper;

import java.io.Serializable;

/**
 * Created by siy on 2016/6/23.
 */
public class ServerStatus implements Serializable {
    private static final long serialVersionUID = 2767708508367266646L;

    private Boolean serverNetworking;
    private Boolean appRunning;
    private Boolean databaseConnecting;

    public Boolean getServerNetworking() {
        return serverNetworking;
    }

    public void setServerNetworking(Boolean serverNetworking) {
        this.serverNetworking = serverNetworking;
    }

    public Boolean getAppRunning() {
        return appRunning;
    }

    public void setAppRunning(Boolean appRunning) {
        this.appRunning = appRunning;
    }

    public Boolean getDatabaseConnecting() {
        return databaseConnecting;
    }

    public void setDatabaseConnecting(Boolean databaseConnecting) {
        this.databaseConnecting = databaseConnecting;
    }

    @Override
    public String toString() {
        return "ServerStatus{" +
                "ServerNetworking=" + serverNetworking +
                ", AppRunning=" + appRunning +
                ", DatabaseConnecting=" + databaseConnecting +
                '}';
    }
}

File: BIPClient/BIPFramework/entity/ServerStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace com.ccf.bip.biz.system.monitor.mapper
{
    [Serializable]
    public class ServerStatus
    {
        private bool serverNetworking;

        public bool ServerNetworking
        {
            get { return serverNetworking; }
            set { serverNetworking = value; }
        }
        private bool appRunning;

        public bool AppRunning
        {
            get { return appRunning; }
            set { appRunning = value; }
        }
        private bool databaseConnecting;

        public bool DatabaseConnecting
        {
            get { return databaseConnecting; }
            set { databaseConnecting = value; }
        }

        public void AllGood()
        {
            serverNetworking = true;
            appRunning = true;
            databaseConnecting = true;
        }
    }
}
